frontend/src/bundles/studio/components/video-menu/components/menu-content/script-content/components/voices-modal/components/voice-card.tsx

```diff
@@ -0,0 +1,50 @@
+import {
+    Card,
+    CardBody,
+    HStack,
+    Text,
+} from '~/bundles/common/components/components.js';
+import { useCallback, useState } from '~/bundles/common/hooks/hooks.js';
+import { IconName, IconSize } from '~/bundles/common/icons/icons.js';
+import { Control } from '~/bundles/studio/components/player-controls/components/control/control.js';
+import { type Voice } from '~/bundles/studio/types/types.js';
+
+type Properties = {
+    voice: Voice;
+    isChecked: boolean;
+    onClick: (voice: Voice) => void;
+};
+
+const VoiceCard: React.FC<Properties> = ({ voice, isChecked, onClick }) => {
+    const [isPlaying, setIsPlaying] = useState(false);
+
+    const handlePlayClick = useCallback((): void => {
+        setIsPlaying((previous) => !previous);
+    }, []);
```

Comment:
You could add event.stopPropagation(); so when the play button is clicked the modal doesnt close automatically.